polkadot/node/network/collator-protocol/src/validator_side/tests/mod.rs

```diff
@@ -756,23 +619,16 @@ fn fetch_one_collation_at_a_time() {
 
```

Comment:
I think because of this check: 
https://github.com/paritytech/polkadot-sdk/blob/eb0ca120671fe7af4d50e863f7d4ac0f553a459d/polkadot/node/network/collator-protocol/src/validator_side/mod.rs#L257

I'll rename the test to avoid confusion.